cmd/admin-user-add.go

```diff
@@ -132,17 +137,19 @@ func (u userMessage) JSON() string {
 	return string(jsonMessageBytes)
 }
 
-// fetchUserKeys - returns the access and secret key
-func fetchUserKeys(args cli.Args) (string, string) {
+// fetchUserParams - returns the access and secret key, user and group id
+func fetchUserParams(args cli.Args) (string, string, string, string) {
 	accessKey := ""
 	secretKey := ""
+	sysUser := ""
```

Comment:
Should them be empty strings or nil? I'd like to not populate this data in database at all. Do we handle empty string and nil the same?

Reply:
I put everywhere omitempty setting, so it shouldn't be in database